Musigwa/SendIT-API
# [WIP] #162034405 prepare database endpoints

### What does this PR do (?)
It sets up the PostgreSQL database environment

### Description of Task to be completed (?)
The following endpoints should be working with the database:

1. GET /
2. GET /parcels
3. GET /parcels/<parcel_id>
4. GET /users/<user_id>/parcels
5. PUT /parcels/<parcel_id>/cancel
6. POST /parcels
7. POST /auth/signup

### How should this be manually tested (?)
After cloning this repo and switch to this branch,

Using any RESTful Client tool (Usually the postman), test every endpoint, make sure to set your Headers as the following
The private Route(s) should return {message: 'access denied'}
The public Route(s) be accessible and working as intended
Key: Content-Type value: application/json

### Any background context you want to provide (?)
I'm using ES6 features in this branch, please be aware of the transpiler (.babelrc file) configurations.

### What are the relevant pivotal tracker stories (?)
162034405
